### PR TITLE
Update to v2.4.3

### PR DESCRIPTION
**Pull request by marcoesters**
Jira ticket: [PKG-574](https://anaconda.atlassian.net/browse/PKG-574)

This requires concurrent uploading with werkzeug and flask, so the `abs.yaml` file needs to stay until everything is approved and ready to be uploaded.

# Changes made

* Update to version 2.4.3
* Remove or change deprecated imports

# Package Information

Home: https://airflow.apache.org/
Development: https://github.com/apache/airflow
Documentation: https://airflow.apache.org/docs/
Change log: https://github.com/apache/airflow/blob/main/RELEASE_NOTES.rst#airflow-2-4-3-2022-11-14

### CODE REVIEW

**Review by varlackc — COMMENTED** [review]
Not sure if we still need the `abs` files
https://github.com/AnacondaRecipes/airflow-feedstock/blob/5edae244336d4364b34f751c212e53852ab69be6/abs.yaml

`Setuptools` must be added as a dependency
https://github.com/apache/airflow/blob/v2-4-stable/setup.py#L34

**Review by skupr-anaconda — COMMENTED** [review]
Missing `setuptools` in host

File: recipe/meta.yaml [hunk 1]
```diff
@@ -278,7 +272,7 @@ outputs:
       imports:
         - airflow
         - cgroupspy
-        - airflow.contrib.task_runner.cgroup_task_runner
+        - airflow.task.task_runner.cgroup_task_runner
       commands:
         - pip check
       requires:
```
[the remark this answers is not in the document]
Comment (marcoesters) [reply recipe/meta.yaml:278]: `airflow` lists `distributed` as an explicit dependency, so I'm not sure if `dask-core` is sufficient: https://github.com/apache/airflow/blob/main/setup.py#L248

**Comment by marcoesters** [comment]
> Not sure if we still need the `abs` files https://github.com/AnacondaRecipes/airflow-feedstock/blob/5edae244336d4364b34f751c212e53852ab69be6/abs.yaml
> 
> `Setuptools` must be added as a dependency https://github.com/apache/airflow/blob/v2-4-stable/setup.py#L34

As I explained in the PR description, the `abs.yaml` file is still needed until merging:

> This requires concurrent uploading with werkzeug and flask, so the abs.yaml file needs to stay until everything is approved and ready to be uploaded.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**